HISTORY.md

```diff
@@ -3,8 +3,12 @@ Pending
 
   * change:
     - requires Node.js 18 or newer
+    - remove deprecated number argument for options
   * breaking:
     - drop support for Node.js 16 and older
+    - response-time no longer supports passing a single number as the `options` argument. Use the `{ digits: <number> }` format instead.
```

Comment:
We might want to do a deprecation notices using depd in 2.x. So it will be easier to migrate later on. WDYT @carpasse ?

Reply:
I add a discussion in private and this is no longer relevant. No change needed